behaviors/TranslatableModel.php

```diff
@@ -208,13 +227,11 @@ protected function loadTranslatableData($locale = null)
             return $this->translatableAttributes[$locale] = [];
         }
 
-        $obj = Db::table('rainlab_translate_attributes')
-            ->where('locale', $locale)
-            ->where('model_id', $this->model->getKey())
-            ->where('model_type', get_class($this->model))
-            ->first();
+        $obj = $this->model->translations->first(function ($value, $key) use($locale) {
```

Comment:
@massimomegistus Please add a space between `use` and `(`:
```suggestion
        $obj = $this->model->translations->first(function ($value, $key) use ($locale) {
```

Reply:
The first() method's arguments should be ($key, $value) ...

ref. https://octobercms.com/docs/services/collections#method-first

Reply:
Actually, looks like the documentation referenced above is wrong the correct order for the callable arguments in the code is ($value, $key):

ref. `vendor/laravel/framework/src/Illuminate/Support/Arr.php` (line 175)

Reply:
@mjauvin I was gonna say... I thought it was always $value, $key. :stuck_out_tongue:

Reply:
Fixed in docs: https://github.com/octobercms/docs/commit/0ff464042c3630ff0026710c2f7376fd9685ea0e